src/mixins/crownConfig.js

```diff
@@ -3,6 +3,7 @@ import trashIcon from '@/assets/trash-alt-solid.svg';
 import pull from 'lodash/pull';
 
 export const highlightPadding = 3;
+import { direction } from '../components/nodes/association/associationConfig';
```

Comment:
I think it looks nicer if you group imports at the top, and you can use the `@` shorthand in the path.